content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,211 @@
+
+In the code above, we imported the `getWalletDetails` and `getLedgerDetails` functions from the get-wallet-details.js and get-ledger-details.js files respectively. Added event listners to the buttons for redirection. Used the `setInterval` method to call the getLedgerDetails function every 4 seconds. (Note: The ledger's estimate close time is 4 seconds, so we are calling the function every 4 seconds to get the latest ledger details. This can be adjusted according to your needs.)
+
+Up next we need to update the HTML and CSS for the home page to give it the XRPL-look and feel. 
+
+7. Copy the contents of the [index.html](_code-samples/build-a-wallet/js/index.html) file to your index.html file.
+
+For this tutorial, we're using basic HTML and pre-loaded CSS files. If you'd like to change the aesthetic, you can replace it with any CSS framework of your choice!
+
+8. Copy the code from the [index.css](_code-samples/build-a-wallet/js/index.css) file to your project's index.css file to use the pre-made stylings.
+
+### Step 3: Creating the Send XRP Page
+
+Till now, we have created the home page that displays the account and ledger details. In this step, we will create the Send XRP page that will allow users to send XRP to another account.
+
+![Send XRP Page Screenshot](img/js-wallet-send-xrp.png)
+
+1. Create a folder named send-xrp in the src directory.
+
+2. Add two files named send-xrp.js and send-xrp.html. You can also create a separate css file depending on your choice. 
+
+3. Copy the contents of the [send-xrp.html](_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.html) file to your send-xrp.html file. The provided HTML code includes three input fields for the destination address, amount, and destination tag, each with their corresponding labels.
+
+4. Now that we have the HTML code, let's add the JavaScript code. In helpers folder, add a new file named submit-transaction.js and copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/helpers/submit-transaction.js", language="js") }}
+
+In the code above, we are using the [submit](https://xrpl.org/submit.html) method to submit the signed transaction to the XRPL. You can read more about the submit method [here](https://xrpl.org/submit.html).
+
+5. Now back to the send-xrp.js file, copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.js", language="js") }}
+
+In the code above, we retrieve elements from DOM, add event listeners, validate inputs and after validation, we call the `submitTransaction` function from the submit-transaction.js file to submit the transaction to the XRPL.
+
+### Step 4: Creating the Transactions Page
+
+Now that we have created the home page and the send XRP page, let's create the transactions page that will display the transaction history of the account.
+
+  1. Account: The account that sent the transaction.
```

Comment:
The screenshot is a bit confusing because the table doesn't display the "Destination" - Could you update the table so:
1. Instead of `Account` it shows `Destination`
2. Somewhere on the screen the user can see their `Account`? 

That way all the key information is on the page. (This may involve modifying the above code snippets to match the implementation)

Reply:
Thanks for pointing it out, forgot to update the screenshot. Fixed.